lib/secrets.rb

```diff
@@ -163,6 +195,21 @@ def read_from_vault(key)
     @vault_v2 ? result.data.fetch(:data).fetch(:vault) : result.data.fetch(:vault)
   end
 
+  def write_to_vault(path, data = {})
```

Comment:
no need for a default
```suggestion
  def write_to_vault(path, data)
```